### ISSUE TITLE
Issues creating certificates with subdomains using Stackpath provider

### ISSUE THREAD

**Issue by JideOsan** [go-acme/lego#1164]
While attempting to create subdomain certificates using Stackpath as a provider, lego successfully creates a TXT record with the name "_acme-challenge" but eventually times out because it is looking for a TXT record with the name "_acme-challenge.sub.domain.com."

```
2020/05/20 00:46:17 [INFO] [sub.domain.com] acme: Obtaining bundled SAN certificate
2020/05/20 00:46:17 [INFO] [sub.domain.com] AuthURL: https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/
2020/05/20 00:46:17 [INFO] [sub.domain.com] acme: Could not find solver for: tls-alpn-01
2020/05/20 00:46:17 [INFO] [sub.domain.com] acme: Could not find solver for: http-01
2020/05/20 00:46:17 [INFO] [sub.domain.com] acme: use dns-01 solver
2020/05/20 00:46:17 [INFO] [sub.domain.com] acme: Preparing to solve DNS-01
2020/05/20 00:46:18 [INFO] [sub.domain.com] acme: Trying to solve DNS-01
2020/05/20 00:46:18 [INFO] [sub.domain.com] acme: Checking DNS record propagation using [8.8.8.8:53 8.8.4.4:53]
2020/05/20 00:46:18 [INFO] Wait for propagation [timeout: 1m0s, interval: 2s]
2020/05/20 00:46:18 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:20 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:23 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:35 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:37 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:44 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:46 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:48 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:50 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:53 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:55 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:57 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:46:59 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:01 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:03 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:05 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:08 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:10 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:12 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:14 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:16 [INFO] [sub.domain.com] acme: Waiting for DNS record propagation.
2020/05/20 00:47:18 [INFO] [sub.domain.com] acme: Cleaning DNS-01 challenge
2020/05/20 00:47:19 [INFO] Deactivating auth: https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/
2020/05/20 00:47:19 Could not obtain certificates:
	error: one or more domains had a problem:
[sub.domain.com] time limit exceeded: last error: NS nsd96q.stackpathdns.net. returned NXDOMAIN for _acme-challenge.sub.domain.com.
```

**Comment by ChrisDeh** [comment]
I'm currently experiencing exact the same Issue with the hetzner DNS Plugin. For the Main Domain Name the DNS Record is properly propagated, but for the Sub Domain it is not.

**Comment by ldez** [comment]
@ChrisDeh the Hetzner provider has been fixed: #1199
